profiler/servlet.go

```diff
@@ -0,0 +1,47 @@
+package profiler
+
+import (
+	"net/http"
+	httppprof "net/http/pprof"
+	"strings"
+
+	"github.com/gorilla/mux"
+
+	"github.com/Shopify/goose/srvutil"
+)
+
+// NewServlet returns a Servlet which can serve pprof requests
+//
+// /debug/pprof/ is the homepage
+// /debug/pprof/ui/ is the same homepage, but where all links point to a UI instead of raw downloads
+// /debug/pprof/*/ are the raw handlers
+// /debug/pprof/ui/*/ are the same handlers, but with a rendered UI
+func NewServlet() srvutil.Servlet {
```

Comment:
```suggestion
func NewServlet(prefix string) srvutil.Servlet {
```

Reply:
The reason it's not configurable is because this is the standard location for this tool and it's what the `go tool pprof http://...` expects.

Reply:
Gross.  I guess my beef is with pprof then.